docs/index.rst

```diff
@@ -35,17 +34,6 @@ Then let's patch all the widely used Python libraries that you are running::
 
 Start your web server and you should be off to the races.
 
-If you want to restrict the set of instrumented libraries, you can either say
```

Comment:
We probably want to keep this section, because it is the easiest way to disable some of the instrumentation (let's say Redis or Postgres integrations happen to report sensitive data, generate too large traces or is buggy).
Could be a section under the `Tutorials` part